lib/hobo_fields.rb

```diff
@@ -23,22 +23,9 @@ module HoboFields
     :integer       => Integer,
     :decimal       => BigDecimal,
     :float         => Float,
-    :string        => String
-  }
-
-  # Provide a lookup for these rather than loading them all preemptively
-
-  STANDARD_TYPES = {
-    :raw_html      => "RawHtmlString",
-    :html          => "HtmlString",
-    :raw_markdown  => "RawMarkdownString",
-    :markdown      => "MarkdownString",
-    :textile       => "TextileString",
-    :password      => "PasswordString",
-    :text          => "Text",
-    :email_address => "EmailAddress",
-    :serialized    => "SerializedObject"
-  }
+    :string        => String,
+    :text          => String
```

Comment:
Is this intentional? I thought `:text` would map to `Text` like it did before?

Reply:
Yea, the `Text` class was a thin rich formatter wrapper around `String` that added a `to_html` formatter.  In the removal of all the rich formatters (because the injection of the special value wrappers was broken and unnecessary with newer versions of rails) `:text` moved back to just being a ruby `String`.  When migrations are generated, the field type is still `text`, but the ruby object is now `String`

Reply:
Ok, got it. Will that change our application code any? We definitely use `:text` fairly often.

Reply:
the ability to use `:text` as a field type is preserved, as long as we're not using `to_html` method [provided by the type](https://github.com/Invoca/hobo_fields/blob/master/lib/hobo_fields/types/text.rb) (which is the only thing it adds), we're fine.

Reply:
Sounds good.